doc/source/dev/development_environment.rst

```diff
@@ -211,10 +211,20 @@ Debugging
 ---------
 
 Another frequently asked question is "How do I debug C code inside NumPy?".
-The easiest way to do this is to first write a Python script that invokes the C
-code whose execution you want to debug. For instance ``mytest.py``::
+First you must obtain the version of gdb higher than 7.0 (which includes an
+embedded Python), secondly make sure the embedded Python is 3.x. To see
+which version of Python is embedded inside your gdb, do the following::
```

Comment:
```suggestion
First, ensure that you have gdb installed on your system with the Python
extensions (often the default on Linux). You can see which version of
Python is running inside gdb to verify your setup::
```
I still don't think the proposed wording captures the original meaning very well. Here's a suggestion for wording that I think captures the original meaning, avoids any potential pitfalls related to gdb or Python version info, and connects to the very nice example that you've included for verifying that gdb is configured appropriately with the Python extensions.

LMK what you think of this wording!

Reply:
Thanks for taking the trouble to help me，I'm struggling at import the python version verifying example smoothly without mention any gdb or python version,  did not find a way to do that(so I mention them all). Your statement is more precise and fluent.